packages/bridge/src/browser.ts

```diff
@@ -0,0 +1,5 @@
+import bridge from './index'
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
You can define property of window with brackets syntax. In this case you don't need to add ts-ignore comment line